Combining datasets (#51)
4TUResearchData-Carpentries/interoperability-climate-sciences: index.md

- semantically understood by humans and machines

> The main message of this lesson is that datasets do not interoperate by themselves; systems interoperate through data that is structured, documented, standardized, and semantically well described. A technically available dataset may still be hard to reuse if its formats, metadata, identifiers, units, vocabularies, and schema are unclear or idiosyncratic. When these elements follow shared standards, the dataset becomes interoperable in the FAIR sense: it can be interpreted and reused across tools, repositories, notebooks, dashboards, cloud workflows, and AI pipelines with far less manual repair.

Reviewer: ```suggestion
The main message of this lesson is that datasets do not interoperate by themselves; systems interoperate through data that are structured, documented, standardized, and semantically well described. A technically available dataset may still be hard to reuse if its formats, metadata, identifiers, units, vocabularies, and schema are unclear or idiosyncratic. When these elements follow shared standards, the dataset becomes interoperable in the FAIR sense: it can be interpreted and reused across tools, repositories, notebooks, dashboards, cloud workflows, and AI pipelines with far less manual repair.
```

At first, the data ecosystem looks rich. She can search across platforms such as [Copernicus Climate Data Store](https://cds.climate.copernicus.eu/), [NASA EarthData](https://www.earthdata.nasa.gov/), the [KNMI Data Platform](https://dataplatform.knmi.nl/), and [4TU.ResearchData](https://data.4tu.nl/). Many datasets are open, downloadable, and described online. Some platforms provide climate model output, others provide satellite products, national weather observations, radar composites, or research datasets deposited by individual research groups.

At 4TU.ResearchData, Ash finds a dataset from the **IRCTR Drizzle Radar (IDRA)**. IDRA is a high-resolution, polarimetric X-band radar developed by TU Delft and located at the Cabauw experimental site in the Netherlands. It is designed to observe low-reflectivity precipitation such as drizzle and light rain within a local observation radius. This makes it highly relevant for Ash’s research question, because drizzle is often difficult to capture consistently across different observation systems.

::::::::::::::::::::::::::::::::: callout 

> [Here](http://ftp.tudelft.nl/TUDelft/irctr-rse/idra/index.html), you can find real time measurements by IDRA. 

Reviewer: ```suggestion
The [real time measurements by IDRA](http://ftp.tudelft.nl/TUDelft/irctr-rse/idra/index.html) are available online.
```
